js/gimmicks/chart.js

```diff
@@ -0,0 +1,111 @@
+(function($) {
+    'use strict';
+
+    var scripturl = 'http://www.chartjs.org/assets/Chart.js';
+
+    var chartGimmick = {
+        name: 'chart',
+        version: $.md.version,
+        once: function() {
+            $.md.linkGimmick(this, 'chart', chart);
+
+            // load the chart js
+            $.md.registerScript(this, scripturl, {
+                license: 'EXCEPTION',
```

Comment:
As to my finding, chartsJs is licensed under MIT license. So we should change 'EXCEPTION' to 'MIT' (MIT is compatible with MDwikis GPL/AGPL).